### ISSUE TITLE
Default value missing

### ISSUE THREAD

**Issue by Rictus** [codemation/easyauth#85]
Hello,

`parse_permissions` is called multiple times in the codebase but is always missing the `default_permissions` argument. I guess it miss a default value ?

https://github.com/codemation/easyauth/blob/7cf8ce8d5b8a5984054d64171aaa0d9e1dbc8eb7/easyauth/server.py#L964

My suggestion:

```  py 
def parse_permissions(self, users, groups, roles, actions, default_permissions=None):
```

**Comment by Rictus** [comment]
This pull request https://github.com/codemation/easyauth/pull/86 aims to solve this

**Comment by codemation** [comment]
Hey @Rictus , good find, looks like this was missed when default permissions were refactored within EasyAuthRouter which does pass the permissions from `self.parent`. Looks good to merge after tests run.

**Comment by codemation** [comment]
Fixed in #86 - thanks for the contribution @Rictus
